pkg/http/api.go

```diff
@@ -698,6 +730,70 @@ func (a *api) getLockStoreWithRequestValidation(reqCtx *fasthttp.RequestCtx) (lo
 	return a.lockStores[storeName], storeName, nil
 }
 
+func (a *api) onStartWorkflow(reqCtx *fasthttp.RequestCtx) {
+	startReq := wfs.StartRequest{}
+
+	wf := reqCtx.UserValue(workflowName).(string)
+	if wf == "" {
+		log.Debug("No workflow, or empty workflow was provided.")
+		return
+	}
+
+	err := json.Unmarshal(reqCtx.PostBody(), &startReq)
+	if err != nil {
+		log.Debug(err)
+	}
+	req := workflows.StartRequest{
+		WorkflowName: startReq.WorkflowName,
```

Comment:
The workflow name be coming from the URL instead of the request body. I see we grabbed it and saved it in the `wf` variable but aren't actually using it.